src/app/api/swarm/jarvis/schema/route.ts

```diff
@@ -31,8 +33,11 @@ export async function GET(request: NextRequest) {
 
     let jarvisUrl = `https://${getSwarmVanityAddress(swarm.name)}:8444`;
     let apiKey = swarm.swarmApiKey;
-    if (process.env.CUSTOM_SWARM_URL) jarvisUrl = `${process.env.CUSTOM_SWARM_URL}:8444`;
```

Comment:
why remove the :8444 here @Rassl ?

if CUSTOM_SWARM_URL is `http://localhost` then all the jarvis, stakgraph, repo2graph endpoints can just use that and add on the correct port